testing/test_aux.out

```diff
@@ -12,12 +12,26 @@ RESULT info protocol.bacnet.version Protocol version: 1
 RESULT pass protocol.bacnet.pic
 RESULT fail connection.mac_oui Manufacturer prefix not found!
 RESULT pass connection.mac_oui
-RESULT skip security.tls.v3
-RESULT skip security.x509
-RESULT fail security.tls.v3
-RESULT fail security.x509
-RESULT pass security.tls.v3
-RESULT pass security.x509
+RESULT fail security.tls.v1
+RESULT fail security.tls.v1.x509
+RESULT fail security.tls.v1_2
+RESULT fail security.tls.v1_2.x509
+RESULT fail security.tls.v1_3
+RESULT fail security.tls.v1_3.x509
+RESULT fail security.tls.v1_2
+RESULT fail security.tls.v1_2.x509
+RESULT fail security.tls.v1_3
+RESULT fail security.tls.v1_3.x509
+RESULT pass security.tls.v1
+RESULT pass security.tls.v1.x509
+RESULT pass security.tls.v1_2
+RESULT pass security.tls.v1_2.x509
+RESULT fail security.tls.v1_3
+RESULT fail security.tls.v1_3.x509
+RESULT pass security.tls.v1_2
+RESULT pass security.tls.v1_2.x509
+RESULT fail security.tls.v1_3
```

Comment:
no skip result (what about not being configured?) and also v1_3 fails all the time?

Reply:
Are any of the faux devices configured to support 1.3?  This is just the test result of the existing device configurations so if they weren't written to support it, it will always fail.